packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -25,6 +26,22 @@ const Redirect = observer(() => {
         is_mobile,
     } = ui;
 
+    // get data from cookies and populate local storage for clients to be logged in
+    const client_accounts = Cookies.get('client.accounts');
+    const active_loginid = Cookies.get('active_loginid');
+
+    if (client_accounts && active_loginid) {
+        localStorage.setItem('client.accounts', client_accounts);
+        localStorage.setItem('active_loginid', active_loginid);
+
+        const domain = getDomainName();
+
+        // remove cookies after populating local storage
+        Cookies.remove('client.accounts', { domain: `.${domain}` });
+        Cookies.remove('active_loginid', { domain: `.${domain}` });
```

Comment:
[M]
```suggestion
        Cookies.remove('active_loginid', { domain });
```

Reply:
https://stackoverflow.com/a/1063760

https://www.rfc-editor.org/rfc/rfc2965

Reply:
Thanks @amam-deriv for the suggestion. It's updated.